blob/blob.go

```diff
@@ -51,7 +50,7 @@ func (b *Blob) Validate() error {
 	if len(b.NamespaceId) != namespace.NamespaceIDSize {
 		return fmt.Errorf("namespace id must be %d bytes", namespace.NamespaceIDSize)
 	}
-	if b.ShareVersion > math.MaxUint8 {
+	if b.ShareVersion > 127 { // see: share.MaxShareVersion
```

Comment:
[optional] should we re-define the constant in this package perhaps at the top of this file? Something like:

```go

// MaxShareVersion is the maximum value a share version can be.
// Note this is copied from share.MaxShareVersion to avoid an import cycle.
MaxShareVersion = 127
```

then here

```suggestion
	if b.ShareVersion > MaxShareVersion {
```

Reply:
Good point, done.